src/components/Modal/Translate.vue

```diff
@@ -35,10 +35,12 @@
 							:disabled="disableFromLanguageSelect"
 							:append-to-body="false" />
 					</div>
-					<textarea ref="input"
-						v-model="input"
+					<NcTextArea ref="input"
+						v-model="inputValue"
+						:label="t('text', 'Text to translate from')"
 						autofocus
-						@input="autosize"
+						input-class="translate-textarea"
+						resize="none"
```

Comment:
It works, but it is incorrect usage of the component. `NcTextArea`, same as other fields in `@nextcloud/vue`, doesn't have `v-model` - it has no defined `model` and doesn't emit `input` event.

It works, because here `v-model` directive catches native event `input` that bubbles with the native event. What it why it was needed to add computed proxy.

Instead, `.sync` modifier should be used on value binding: `:value.sync="input"`.